### PR TITLE
Add Trifinger Simulation

### PR DESCRIPTION
**Pull request by mposa**
Adds an example using the Trifinger platform. Of note,the mesh files here aren't small (13.6 MB total).

TODO before merging: I'll also add an example of a Python controller

---
This change is [Reviewable](https://reviewable.io/reviews/dairlab/dairlib/278)

### CODE REVIEW

**Comment by mposa** [comment]
This example should be good to ship. Includes a python controller and demonstrates how to log the results of a simulation into a numpy array.

**Comment by mposa** [comment]
This will need to wait on #274 to merge cleanly

File: examples/trifinger/simulate_trifinger.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+class TrifingerDemoController(LeafSystem):
+  def __init__(self, plant):
+    LeafSystem.__init__(self)
+
+    self.plant = plant
+        
+    # Input is state, output is torque (control action)
+    self.DeclareVectorInputPort("x", plant.num_positions() +
+     																 plant.num_velocities())
+    self.DeclareVectorOutputPort("u", plant.num_actuators(),
```
Comment (Brian-Acosta) [inline examples/trifinger/simulate_trifinger.py]: Formatting for these two calls

**Review by Brian-Acosta — COMMENTED** [review]
Just the one formatting issue/typo

**Review by mposa — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dairlab/dairlib/278)* status: 0 of 94 files reviewed, 1 unresolved discussion (waiting on @Brian-Acosta)

___
*[examples/trifinger/simulate_trifinger.py, line 16 at r1](https://reviewable.io/reviews/dairlab/dairlib/278#-MuWZqbBARqDWLoX4coc-r1-16:-MuuuxrN4uIma9DeGJi5:b-896fix) ([raw file](https://github.com/dairlab/dairlib/blob/995bc51f38f9435e4a7eb1004efa75141530cee8/examples/trifinger/simulate_trifinger.py#L16)):*
<details><summary><i>Previously, Brian-Acosta (Brian Acosta) wrote…</i></summary><blockquote>

Formatting for these two calls
</blockquote></details>

Done.

**Pull request merged**